### DISCUSSION TITLE
Weird error causing 500 error

### DISCUSSION THREAD

**Discussion by JigolKa** [vercel/community#3539] [Help] [answered]
### Summary

Hi, 
I got a weird error complaining about having too many files open, I don't really know what could have caused this.
Here's the full log:
```
[Error: EMFILE: too many open files, open '/var/task/node_modules/lucide-react/dist/esm/icons/shrub.mjs'] {
  errno: -24,
  code: 'EMFILE',
  syscall: 'open',
  path: '/var/task/node_modules/lucide-react/dist/esm/icons/shrub.mjs',
  page: '/tweet/clkve6v3l000awz6kudt2pfic'
}
```

Works fine on my machine

### Example

https://twitter-clone-jigolka.vercel.app/

### Steps to Reproduce

I have absolutely no idea how to reproduce this. Just click on the first post in the demo, and you should get a 500 code

**Comment by amyegan** [comment]
Hey there. This EMFILE error means that the build work exceeds the Vercel builder limits. You may need to change the icon import in your file. Someone else had a similar problem with a different icon package a couple of months back: https://github.com/vercel/community/discussions/2865

Something else that may help is wrapping related sections of code in [try...catch statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) to log additional error info. That should give you more info to help you locate source of the error.

**Comment by JigolKa** [comment, marked as answer]
Problem solved, I had to install this specific version: `lucide-react@0.263.2-beta.0`
More here: [https://github.com/lucide-icons/lucide/issues/1457](https://github.com/lucide-icons/lucide/issues/1457)

Thanks @amyegan
